code/modules/mob/mob.dm

```diff
@@ -495,6 +495,10 @@ var/global/obj/screen/fuckstat/FUCK = new
 	var/obj/item/W = get_active_hand()
 	if(istype(W))
 		equip_to_slot_if_possible(W, slot)
+	else
+		W = get_item_by_slot(slot)
+		if(W)
+			W.attack_hand(src)
```

Comment:
You know, it just occurred to me that I didn't test this with just that bit there. When I get home later I'll test it.